### ISSUE TITLE
Build Squirrel has some error on macOS

### ISSUE THREAD

**Issue by sunzhenyucn** [rime/squirrel#205]
### The Build Environment: 
- macOS version: macOS Hign Sierra 10.13.4  
- XCode Command Line Tools version: 9.3 Update time: Apr 2, 2018  
- CMake version: 3.11.1  
- Boost version: 1.67.0_1  
- Git version:  2.15.1  

### The error content:  
```shell
cp -R thirdparty/src/gtest/include/gtest thirdparty/include/
cp thirdparty/src/gtest/build/libgtest*.a thirdparty/lib/
/Library/Developer/CommandLineTools/usr/bin/make -C librime -f Makefile.xcode release
cmake . -Bxbuild -GXcode -DBUILD_STATIC=ON -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON
CMake Error:
  Xcode 1.5 not supported.


CMake Error: Could not create named generator Xcode
```  
### The error content screenshot:  

![rime-error-screenshot.jpg](https://i.loli.net/2018/04/20/5ad95cfe0f7c8.jpg)  

**If anyone know how fix this error, please tell me, thanks!!!**

**Comment by KochamCie** [comment]
surprise mother fucker

**Comment by sunzhenyucn** [comment]
@KochamCie 23333

**Comment by sunzhenyucn** [comment]
fixed, see [#157](https://github.com/rime/squirrel/issues/157)
